stdlib/public/TensorFlow/Gradients.swift

```diff
@@ -589,6 +589,15 @@ extension Tensor where Scalar : TensorFlowFloatingPoint {
     })
   }
 
+  @inlinable
+  func _vjpMean(alongAxes axes: Tensor<Int32>) -> (Tensor, (Tensor) -> Tensor) {
+    let value = mean(alongAxes: axes)
+    return (value, { [shape = shapeTensor,
+            count = Raw.gather(params: shapeTensor, indices: axes).product()] in
```

Comment:
Could you please declare `count` as a local variable under `let value`?

Reply:
Done.